### ISSUE TITLE
Homebridge Crashing with v1.0.5

### ISSUE THREAD

**Issue by washcroft** [ebaauw/homebridge-soma#9]
Not sure what's going on here - it only seems to happen whilst running as a child bridge.

```
[11/9/2021, 8:53:44 AM] [SOMA] warning: heartbeat 1, drift 459

[11/9/2021, 8:53:44 AM] [SOMA] warning: d415ddd5248a: request 2: disconnected unexpectedly


/homebridge/node_modules/homebridge-soma/lib/SomaPlatform.js:154

            delegate.destroy()

                     ^

TypeError: delegate.destroy is not a function

    at SomaClient.<anonymous> (/homebridge/node_modules/homebridge-soma/lib/SomaPlatform.js:154:22)

[11/9/2021, 8:53:44 AM] [SOMA] goodbye

[11/9/2021, 8:53:44 AM] [SOMA] Child bridge process ended

[11/9/2021, 8:53:44 AM] [SOMA] Process Ended. Code: 1, Signal: null

[11/9/2021, 8:53:51 AM] [SOMA] Restarting Process...

Initializing HAP-NodeJS v0.9.6...

[11/9/2021, 8:53:52 AM] [SOMA] Launched child bridge with PID 499148

[11/9/2021, 8:53:52 AM] Registering platform 'homebridge-soma.Lib'
```

**Comment by ebaauw** [comment]
Looks like I was a bit overzealous in refactoring `BleClient`.

The issue is not related to child bridges, but it occurs when a new accessory is created.  If you moved your existing installation to a child bridge, without copying/renaming `cachedAccessories`, Homebridge SOMA in the child bridge would create new accessories.

**Comment by ebaauw** [comment]
Fixed in v1.0.6.

**Comment by paulocpcorrea** [comment]
I'm having the same problem. I'm running on version 1.0.7. Can you help me?

`[14/11/2021, 08:47:58] [SOMA] warning: d52b84bf36c9: request 2: disconnected unexpectedly
[14/11/2021, 08:48:04] [SOMA] warning: d52b84bf36c9: request 2: disconnected unexpectedly
[14/11/2021, 08:48:10] [SOMA] warning: d52b84bf36c9: request 2: disconnected unexpectedly
[14/11/2021, 08:48:16] [SOMA] warning: d52b84bf36c9: request 2: disconnected unexpectedly
[14/11/2021, 08:48:22] [SOMA] warning: heartbeat 371, drift 867
[14/11/2021, 08:48:22] [SOMA] warning: d52b84bf36c9: request 2: disconnected unexpectedly
[14/11/2021, 08:48:28] [SOMA] warning: d52b84bf36c9: request 2: disconnected unexpectedly
[14/11/2021, 08:48:34] [SOMA] warning: heartbeat 383, drift 532
[14/11/2021, 08:48:34] [SOMA] warning: d52b84bf36c9: request 2: disconnected unexpectedly
[14/11/2021, 08:48:40] [SOMA] warning: heartbeat 389, drift 359
[14/11/2021, 08:48:40] [SOMA] warning: d52b84bf36c9: request 2: disconnected unexpectedly
[14/11/2021, 08:48:46] [SOMA] warning: heartbeat 395, drift 305
[14/11/2021, 08:48:46] [SOMA] warning: d52b84bf36c9: request 2: disconnected unexpectedly
[14/11/2021, 08:48:52] [SOMA] warning: d52b84bf36c9: request 2: disconnected unexpectedly
[14/11/2021, 08:48:58] [SOMA] warning: d52b84bf36c9: request 2: disconnected unexpectedly
[14/11/2021, 08:49:04] [SOMA] warning: heartbeat 413, drift 273
[14/11/2021, 08:49:04] [SOMA] warning: d52b84bf36c9: request 2: disconnected unexpectedly
[14/11/2021, 08:49:10] [SOMA] warning: d52b84bf36c9: request 2: disconnected unexpectedly
[14/11/2021, 08:49:16] [SOMA] warning: d52b84bf36c9: request 2: disconnected unexpectedly
[14/11/2021, 08:49:22] [SOMA] warning: d52b84bf36c9: request 2: disconnected unexpectedly
[14/11/2021, 08:49:28] [SOMA] warning: d52b84bf36c9: request 2: disconnected unexpectedly
[14/11/2021, 08:49:33] [SOMA] warning: d52b84bf36c9: request 2: disconnected unexpectedly
[14/11/2021, 08:49:40] [SOMA] warning: heartbeat 449, drift 265
[14/11/2021, 08:49:40] [SOMA] warning: d52b84bf36c9: request 2: disconnected unexpectedly
[14/11/2021, 08:49:46] [SOMA] warning: heartbeat 455, drift 392
[14/11/2021, 08:49:46] [SOMA] warning: d52b84bf36c9: request 2: disconnected unexpectedly
[14/11/2021, 08:49:52] [SOMA] warning: d52b84bf36c9: request 2: disconnected unexpectedly
[14/11/2021, 08:49:58] [SOMA] warning: d52b84bf36c9: request 2: disconnected unexpectedly
[14/11/2021, 08:50:04] [SOMA] warning: d52b84bf36c9: request 2: disconnected unexpectedly`

It started giving this error after I updated Node.js to 16.13 and the plugin to 1.0.7. Before, it worked normally.
